PE/__init__.py

```diff
@@ -1033,7 +1033,7 @@ def getSignCertInfo(self):
             cbytes = pyasn1.codec.der.encoder.encode( i['certificate'] )
 
             iparts = []
-            for rdnsequence in i["certificate"]["tbsCertificate"]["issuer"]:
+            for _, rdnsequence in i["certificate"]["tbsCertificate"]["issuer"].items():
```

Comment:
This is actually a miscellaneous bug fix. `i["certificate"]["tbsCertificate"]["issuer"]` is a dictionary, so the previous way was just iterating over the keys and not the values, so we weren't actually getting any issuer or subject data.

We can change it to just a call to values(), but items() is a generator, and I prefer to yield values up instead of grabbing the whole list all at once.

Reply:
love the generator better.  works for me.